content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/aging.rst

```diff
@@ -1,98 +1,224 @@
+  change in the product's inventory valuation method.
+- :guilabel:`Remaining Value`: the value of this product's current stock levels for this particular
+  stock move, after demand has been accounted for. Along with :guilabel:`Remaining Quantity`, this
+  field can be especially helpful for :abbr:`FIFO (First In, First Out)` and
+  :abbr:`AVCO (Average Cost)` accounting, as they convey which stock came into a warehouse first and
+  the value of said stock.
+
+.. tip::
+   Some of these settings may not appear unless first enabled in Odoo's **:doc:`Settings
+   <../../../general>`** application.
+
+.. image:: aging/stock-valuation-report.png
+   :alt: Stock valuation report.
+
+.. _inventory/warehouses_storage/svl:
+
+Stock valuation layers (SVLs)
+-----------------------------
+
+Each line item in the :guilabel:`Stock Valuation` report represents a record in Odoo's system known
+as a *stock valuation layer (SVL)*. :abbr:`SVLs (stock move layers)` are generated when products
+move in a way that impacts their stock valuation. Specifically, the stock moves that generate
+:abbr:`SVLs (stock move layers)` are warehouse receipts, deliveries, dropshipping orders, and
+dropshipping returns. These stock moves must first be validated (by clicking the
+:guilabel:`Validate` button) for the :abbr:`SVL (stock move layer)` to be created.
+
+If a product's inventory valuation method changes on the product form, new line items are generated
+on the :guilabel:`Stock Valuation` report to reflect the resulting :abbr:`SVLs (stock move layers)`.
+For example, if the valuation method changes from *standard price* to either *:abbr:`AVCO (Average
+Cost)`* or *:abbr:`FIFO (First In, First Out)`* accounting, *revaluation entries* will be
+automatically posted to reflect the change in pricing for goods that remain in stock. One entry will
+be negative to "remove" the old pricing, and the second entry will be positive to record the new
+pricing. These entries are connected to journal entries in Odoo's **Accounting** app.
+
+Below is an example of what the :guilabel:`Stock Valuation` table shows when a few stock moves have
+occurred for a product using standard price accounting.
+
+.. image:: aging/before-val-method-change.png
+   :alt: Stock valuation table in standard price accounting.
+
+Conversely, the following image depicts what the *Stock Valuation* Report table might look like
+after a product has switched from standard price to :abbr:`FIFO (First In, First Out)` accounting.
+
+.. image:: aging/after-val-method-change.png
+   :alt: Stock valuation table after switching from standard price to FIFO accounting.
+
+.. example::
+   The :guilabel:`Remaining Value` and :guilabel:`Remaining Quantity` fields are derived from what
+   occurs at the :abbr:`SVL (stock move layer)` level in Odoo and, as such, are better understood
+   with an example.
+
+   `Frankie's Consignment Shop` buys sweaters at the cost, or :guilabel:`Unit Value`, of `5.00`
```

Comment:
The literal formatting on the company name seemed odd to me, I'd suggest removing the formatting– but totally up to you!
```suggestion
   Frankie's Consignment Shop buys sweaters at the cost, or :guilabel:`Unit Value`, of `5.00`
```